Include/dpsim/pybind/DPComponents.h

```diff
@@ -0,0 +1,31 @@
+/* Copyright 2017-2020 Institute for Automation of Complex Power Systems,
+ *                     EONERC, RWTH Aachen University
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *********************************************************************************/
+#pragma once
+
+#include <pybind11/pybind11.h>
+#include <pybind11/complex.h>
+#include <pybind11/stl.h>
+#include <pybind11/functional.h>
+#include <pybind11/eigen.h>
+
+#include <dpsim/Simulation.h>
+#include <dpsim/RealTimeSimulation.h>
+#include <cps/IdentifiedObject.h>
+#include <cps/CIM/Reader.h>
+#include <DPsim.h>
```

Comment:
@JTS22 would it be possible to move these dpsim includes to the cpp files?

Reply:
yes, did that in 5c0d595dbba0a90d0a5e84122cf42474193f1c42